modules/components/src/Aggs/aggComponentsMap.js

```diff
@@ -7,16 +7,17 @@ import { fieldInCurrentSQON } from '../SQONView/utils';
 const composedTermAgg = ({ sqon, onValueChange, ...rest }) => (
   <TermAgg
     handleValueClick={({ generateNextSQON, value, field }) => {
+      let nextSQON = generateNextSQON(sqon);
+      const active = fieldInCurrentSQON({
+        currentSQON: nextSQON ? nextSQON.content : [],
```

Comment:
super small thing :) this can be written with optional chaining as `nextSQON?.content || []`